### PR TITLE
Update to Tricia (alternate approach)

### LINKED ISSUE #347: Update to Mint Tricia 19.3
New base release expected within weeks. Beta test branch available here: https://github.com/ripleymj/cs-vm-build/tree/tricia-beta-test

### LINKED ISSUE #359: packer build broken with VirtualBox 6.1: VBoxManage: error: Unknown option: --clipboard
It looks like the option has now become
`[--clipboard-mode disabled|hosttoguest|guesttohost|bidirectional]`.

Looks like we'll have to decide whether to build with 6.0 or 6.1 for Spring 2020.

### PR DESCRIPTION
**Pull request by ripleymj**
Per some conversation, this cleans up the beta process a bit and would replace PR #360.

Closes #347
Closes #359

### CODE REVIEW

**Comment by kylelaker** [comment]
This is my preferred approach to adding the beta support

**Comment by ripleymj** [comment]
Pretty sure I've always used master to build the images, unless I'm testing a PR. Branching the release usually happens after we've built an image, and got master working against it.

File: packer/beta-vars.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "semester": "Sp20",
+  "mint_version": "19.3",
+
+  "git_repo": "https://github.com/jmunixusers/cs-vm-build",
```
Comment (kylelaker) [inline packer/beta-vars.json]: Since these are the defaults, do they need to be overriden in this file?

Comment (ripleymj) [reply packer/beta-vars.json]: I fiddle with the repo and branch enough during beta testing that it would be helpful to have it present.

**Pull request merged**